packages/storybook/src/nimble/anchor-tabs/anchor-tabs.stories.ts

```diff
@@ -79,20 +85,25 @@ const metadata: Meta<AnchorTabsArgs> = {
 export default metadata;
 
 export const anchorTabs: StoryObj<AnchorTabsArgs> = {
+    // prettier-ignore
     render: createUserSelectedThemeStory(html`
-    <${anchorTabsTag} activeid="${x => x.activeid}">
-        ${repeat(
-        x => tabSets[x.tabsType] as AnchorTabArgs[],
-        html<AnchorTabArgs>`
+        <${anchorTabsTag} activeid="${x => x.activeid}">
+        ${repeat(x => tabSets[x.tabsType] as AnchorTabArgs[], html<AnchorTabArgs>`
             <${anchorTabTag}
                 ?disabled="${x => x.disabled}"
                 id="${x => x.id}"
                 href="${x => x.href}"
             >
                 ${x => x.title}
             </${anchorTabTag}>
-        `
-    )}
+        `)}
+        ${when(x => x.tabsType === ExampleTabsType.simpleTabsWithToolbar, html`
+            <${tabsToolbarTag}>
+                <${buttonTag} appearance="ghost">Toolbar Button</${buttonTag}>
+                <${buttonTag} appearance="ghost" slot="end">Toolbar Button 2</${buttonTag}>
+                <${buttonTag} appearance="ghost" slot="end">Toolbar Button 3</${buttonTag}>
+            </${tabsToolbarTag}>
+        `)}
     </${anchorTabsTag}>
     `),
     argTypes: {
```

Comment:
Can you update the labels in tabsType below to give pretty display version like the others:
![image](https://github.com/user-attachments/assets/bf3ca261-66eb-47d3-921f-a662f0e21f22)